### PR TITLE
ARUHA-1978 Be able to specify commit timeout less than 60 seconds

### PR DESCRIPTION
**Pull request by antban**
ARUHA-1978 Be able to specify commit timeout less than 60 seconds

### CODE REVIEW

**Comment by antban** [comment]
:+1:

**Comment by lmontrieux** [comment]
👍

**Comment by lmontrieux** [comment]
Deploy validation please.

**Comment by lmontrieux** [comment]
deploy validation please

File: src/main/java/org/zalando/nakadi/view/UserStreamParameters.java [hunk 1]
```diff
@@ -25,21 +25,25 @@
...
     @JsonCreator
     public UserStreamParameters(@JsonProperty("batch_limit") @Nullable final Integer batchLimit,
                                 @JsonProperty("stream_limit") @Nullable final Long streamLimit,
                                 @JsonProperty("batch_flush_timeout") @Nullable final Integer batchFlushTimeout,
                                 @JsonProperty("stream_timeout") @Nullable final Long streamTimeout,
                                 @JsonProperty("stream_keep_alive_limit") @Nullable final Integer streamKeepAliveLimit,
                                 @JsonProperty("max_uncommitted_events") @Nullable final Integer maxUncommittedEvents,
-                                @JsonProperty("partitions") @Nullable final List<EventTypePartition> partitions) {
+                                @JsonProperty("partitions") @Nullable final List<EventTypePartition> partitions,
+                                @JsonProperty("commit_timeout") final Long commitTimeoutSeconds) {
```
Comment (v-stepanov) [inline src/main/java/org/zalando/nakadi/view/UserStreamParameters.java]: `@Nullable`

**Comment by antban** [comment]
@lmontrieux I fixed the bug you found

**Comment by lmontrieux** [comment]
👍

**Comment by antban** [comment]
:+1:

**Comment by antban** [comment]
:+1:

**Comment by antban** [comment]
:+1:

**Comment by v-stepanov** [comment]
👍

**Pull request merged**